index.js

```diff
@@ -0,0 +1,37 @@
+function calculateSwitch(numbers) {
+  switch (numbers[1]) {
+    case '+':
+      return numbers[0] + numbers[2]
+    case '-':
+      return numbers[0] - numbers[2]
+    case '*':
+      return numbers[0] * numbers[2]
+    case '/':
+      return numbers[0] / numbers[2]
+    default: return NaN
+  }
+}
+function isUsableValue(value) {
+  const operators = ['+', '-', '*', '/']
+
+  return isNum(value) || operators.includes(value)
+}
+function isNum(value) {
+  return !isNaN(value)
+}
+function isEmptyStringValue(value) {
+  return value === ''
+}
```

Comment:
I would call this in the `isUsableValue` function rather than the main calculate function. Technically not being an empty string is part of being usable so it belongs there and would simplify the if in your calculate function.